docs/source/common/lightning_module.rst

```diff
@@ -907,8 +952,8 @@ The current step (does not reset each epoch)
 
 hparams
 ~~~~~~~
-The arguments saved by calling ``save_hyperparameters`` passed through ``__init__()``
- could be accessed by the ``hparams`` attribute.
+The arguments saved by calling :meth:`~pytorch_lightning.core.mixins.HyperparametersMixin.save_hyperparameters`
```

Comment:
```suggestion
The arguments saved by calling ``save_hyperparameters``
```
I see you did multiple changes to this format, I prefer the existing one. Imo, user doesn't need to know the reference

Reply:
aren't references better? if they want to have a look at it.. then it's quick, no search required.